docs/csharp/programming-guide/classes-and-structs/passing-reference-type-parameters.md

```diff
@@ -14,7 +14,7 @@ author: "BillWagner"
 ms.author: "wiwagn"
 ---
 # Passing Reference-Type Parameters (C# Programming Guide)
-A variable of a [reference type](../../../csharp/language-reference/keywords/reference-types.md) does not contain its data directly; it contains a reference to its data. When you pass a reference-type parameter by value, it is possible to change the data pointed to by the reference, such as the value of a class member. However, you cannot change the value of the reference itself; that is, you cannot use the same reference to allocate memory for a new class and have it persist outside the block. To do that, pass the parameter using the [ref](../../../csharp/language-reference/keywords/ref.md) or [out](../../../csharp/language-reference/keywords/out.md) keyword. For simplicity, the following examples use `ref`.  
+A variable of a [reference type](../../../csharp/language-reference/keywords/reference-types.md) does not contain its data directly; it contains a reference to its data. When you pass a reference-type parameter by value, it is possible to change the data pointed to by the reference, such as the value of a class member. However, you cannot change the value of the reference itself; that is, you cannot use the same reference to allocate memory for a new class and have it persist outside the block. To do that, pass the parameter using the [in](../../../csharp/language-reference/keywords/in.md), [ref](../../../csharp/language-reference/keywords/ref.md) or [out](../../../csharp/language-reference/keywords/out.md) keyword. For simplicity, the following examples use `ref`.  
```

Comment:
A bunch of nits unrelated to your PR:
- "the data pointed to by the reference" becomes confusing, since the reference already is a pointer to the object. I'd interpret this to say that you are changing the value of the reference itself, which is what the next sentence says you cannot do. I think it would be less confusing to say "the data belonging to the referenced object" or some such thing. 
- "that is" is a general statement, but this is a specific example; "for example" would be better here.
- "block" is a method in this context; it would be best to describe it as a method.